quinn-proto/src/token.rs

```diff
@@ -77,16 +78,23 @@ pub(crate) struct InvalidRetryTokenError;
 
 /// Retry or validation token
 pub(crate) struct Token {
+    /// Randomly generated value, which must be unique, and is visible to the client
+    rand: u128,
```

Comment:
I don't love the `rand` name for this. Would `nonce` make sense as a name for this?